content/release-notes/1.17.0.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2020-07-13T08:00:00-07:00
+linktitle: "1.17.0"
+title: "1.17.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* GitOps - Added Github Enterprise support.
+* Snapshots - Added a "Internal Storage (Default)" option to the providers list for KURL embedded clusters. Which will switch back to the default configuration that velero was set up with when the app was installed.
+* Added `when` field support for config groups.
+{{</features>}}
+
+{{<changes>}}
+* Kotsadm - New Support bundles will be shown at the top of the list instead of the bottom.
+* Kotsadm - Showing warnings on the Version History page in the Admin Console when installing an application with invalid YAML.
+* Kotsadm - Added a page title for the 'View files' and 'Config' tabs.
+* Removed the KOTS metadata that was added to all application manifests. Which included the local application and the version sequence injected as annotations to all deployed manifests.
```

Comment:
Why did we do this?

Reply:
It was force restarting pods for customers, so we decided to get rid of them for the time being and come with a better solution in the future. Also, these annotations are not used at all yet.

Reply:
I'd then put this under the bug section, and say: 
```
Fixed an issue where pods were force-restarted because of KOTS metadata in application manifests. Fixed by ensuring that the system no longer adds KOTS metadata to these manifests. 
```

Reply:
I think this should stay as a change. Removing this could possibly have adverse effects on an app if they were using undocumented behavior. Specifically this was removed ONLY for gitops enabled clusters (which should be called out)

Reply:
In that case, I'd advocate it being both a change and a bug. I'd also call out GitOps enabled clusters. 

```suggestion
* Removed the KOTS metadata that was added to all application manifests. For GitOps enabled clusters, this metadata could result in situations where pods were incorrectly force-restarted. 
```

Reply:
This is not correct.  We only added it to GitOps clusters.

Reply:
Ok, tried to update accordingly.